ext/har/types.go

```diff
@@ -0,0 +1,365 @@
+// Original implementation from abourget/goproxy, adapted for use as an extension.
+// HAR specification: http://www.softwareishard.com/blog/har-12-spec/
+package har
+
+import (
+    "bytes"
+    "io"
+    "io/ioutil"
+    "log"
+    "net"
+    "net/http"
+    "net/url"
+    "strings"
+    "time"
+)
+
+var startingEntrySize int = 1000
```

Comment:
Instead of a global variable, add a
const startingEntrySize = 1000
inside `makeNewEntries`